### DISCUSSION TITLE
meta-hours #1

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#1424] [Meta-Hours]
- [[meta-hours-composition]]

###  Sharing boring numbers like 
- Over .5T bandwidth for Netlify for past 30 days
- Over 1T served files from Pinata 
- 675build minutes
- 10k+ unique visitors
- Quarter Milion pageviews (261k)
- 106 Active Pull Requests, by 16 people
- 266 Active Issues
- Backlog of over 420 issues which is good if I die but not good to push product fast enough, that's why we are still trying hire great developer as our intern
- recently we've been listed on awesome-vue and soon we will be listed on awesome-nuxt so we bet more Vue/Nuxt developers would know about us
- kodadot.xyz alone had 1,463 builds.
- Your longest streak was 32 days with at least one build.
- You have 1 site using Netlify Functions that have been called 6,688 times.

### Beta.kodadot.xyz
- Seems as we are making new upgrades and bug fixes to old legacy app at kodadot.xyz there are still some things missing to our new beta
- Hard to estimate anticipated progress as some stuff we're pushed to legacy version for reason have it there and we need still to re-implement it into new version.
- You can try on your own risk beta.kodadot.xyz, it should be significantly faster, it should have over 80% functionality from legacy app already implemented
- You can most probably also mint from that version which should be working nowadays
- We will introduce there new graphs, where thanks to RhaiCode we will have completely new chart engine where we can start adding new experiments for as chart extension. 
- We will added to see buy events, list events and median to see average price
- Among first chart extension we've tried implement H-spread, IQR, which should tell you price distribution per collection so you can see bottom and upper level. So that's coming up next to better ease your decision about collection
- Another significant thing why it takes so long is that reimplementation from all legacy codebase we had to decide which functionality we need to drop, which keep and which extend prior where users could further experiment with us if it works or not for them.

### KodaVerse
- We've had quick sync with Roncho and I believe he will be able to make it and show up
- We discussed way to start working releasing v0 implementation which would be using fy_iceworld_gallery to show things on the wall, showing 3d models on the islands and adding portals/doors so users can drive through various galleries like you do with collections now.
- Next stage is to open doors for artits adding their own world, adding custom voice narration for their gallery, so we can emulate trully right exhibition and ways to incentivise foot traffic for future collectors and much more.
- Right now we are trully focusing on delivering working stuff so we can be listed in other XR spatial worlds and we have really strong peopel joining us on background, like people who done something similiar, they exited company, one of them already acquired by strong company out there. Literally, same as KodaDot, just one economy cycle back, inb4 till NFT wasn't existing and it was just boring web1 and web2 

### Logo design
- Yeah, we've started as rewritte of Vue polkadot dotapps and we had really basic logo which was just reflecting basic Polkadot, then we upgraded thanks to Anette to new one which should just reflect switches on web forms, probably these days no one gets it, hah and then thanks to Xylodrone we had one square like logo which more reflecting our thinking out of box naturally and framing of jpegs and one can find there that our logo tends to go out of frame which means to be spatial. We are still thinking this was such a great and simple logo.

- Nowadays logo would be more reflecting our name, Koda, which should have meaning in japanese happy rice paddy and more logo looks like from Japanese Kanji with Manga texture. 
- KodaDot v3 logo press kit pack
- Let's Xylodrone talk

### Community efforts
- Recently we've quite miss right community lingo in Stickers. 
- We would like to firstly thanks to amazing work done by Curu who was working on long-term basis brining emojis and stickers to our KodaDot while we're existing on Telegram and further
- Nowadays we are thinking put out contest for creating animated stickers, stickers and animated emojis for KodaDot Discord  
- damsky can speak about more

### Meta Mobile App
- Previous week we've got together with Martin Jesnen and Shady/Shekohex where they can help us deliver first iteration of v0 app so others can start using it and browse briliant art pieces published on KodaDot 
- In later stages we will be exploring possibilities which features should not be missing

### Want to keep the pace with us?
- Best is to visit our issues in repository where I'm droping most of stuff where I felt inspired and might fit for future version of KodaDot 

### Landing 
- Let's peak on landing, Xylodrone and JK are in the charge of this and I would like to show you peak what we are thinking what would be appearing on Metaprime.network
- Hence we didn't finish first version 1, for obvious reason we've started working on second iteration where we want to fold less relevant informations for fast scrollers, skimmers and others incoming members of MetaPrime and KodaDot 
- Goal is to have futuristic tuned website in fashion like cassetepunk, retro 80s green, like from wargames where people can start building better worlds with MetaPrime
- We have something on background cooking what will be MetaPrime up to, hence don't want to take this long, we will be showing one by one once we will be growing with upcoming investments from our investors

### KodaForest
- We've successfully planted 330 trees in Portugal thanks to Pedro and his amazing initiative RMTerra. I would like to show up some love in upcoming episodes of MetaHours and make it dedicated talk how NFTs can improve climate change by driving sales, revenues and various game plays ppl can support planet through owning NFTs
- We have more partners lining up on this tune, so want to wait a bit whenever all information could be shared publicly, even from our side so all beautiful partnership supporting planet Earth will come true and all we will feel connected 👀

### Koda-Token
- [[koda-token]]
- With paying out bounties we came to realization that some developers and users would like to have exposure on product with their past, present and future contribution. There we put in the pipeline start thinking on the right way to gain this exposure. Right now we are in phase where we are thinking what should be meaningful distribution for token, who should get and how much. Definitely want to avoid future gamers albeit intense contributions are welcome. We would like to put emphasis on incentivize future contirbutions and reward those past. Probably our model is currently closed to Gitcoin one, which could create healthy set of maintainers and future governance could be breeze, iterate quickly and fast tracked for new changes.

- We are in chat with few tokenomics studios who could help us share spotlight on particular insights we might miss and I agree, we are humans, we might have blindspots and our intent is reflect on best what we are doing right now.

- Goal is to lower our expense through $KSM hence it was sponsored out of our pocket to drive KodaDot development
- It would help attract high positioned whales who would like to take care of product and create sense of ownership stake in stuff they are using to gain higher exposure to future returns. So I bet they would have same intention to keep KodaDot afloat as we are in current stage give blizzard insights for each collections and next advanced insights are in pipeline as my hobby is to come over, find insightful inspiration from other products and trying retrofit it to our KodaDot. Similiar like Spotify has tons of inspiration for me where I'm trying reverse engineer algorhytms which might have huge impact for quality of audience and people around KodaDot

- Following all this, we found governance channel and let discussion spark there, people who would like be part of, definitely show up and we want also merceneries around but contrary, we would like support counterfactual value of each contributor as it's hard to estimate we need find consensus in right estimation of everyone participation to have share and sense of ownership KodaDot

- Open discussion? but maybe not as best would be having opinions written on discussion board for fair of others

**Comment by ins0mn1a** [comment]
Kodadot Meta Hours #1 recording.
https://www.youtube.com/watch?v=NektwvmuTaU

**Reply by yangwao** [reply]
> Kodadot Meta Hours #1 recording. https://www.youtube.com/watch?v=NektwvmuTaU

Payout sent for recording! 
https://kusama.subscan.io/extrinsic/0x1a4c9df158e645bcf089011091ba1d403182caebea384cbdf5736d682efb498b
